sentinelsat/scripts/cli.py

```diff
@@ -144,12 +141,11 @@ def cli(user, password, geometry, start, end, uuid, name, download, md5, sentine
             outfile.write(gj.dumps(footprints_geojson))
 
     if download is True:
-        product_infos, failed_downloads = api.download_all(products, path, checksum=md5)
-        if md5 is True:
-            if len(failed_downloads) > 0:
-                with open(os.path.join(path, "corrupt_scenes.txt"), "w") as outfile:
-                    for failed_id in failed_downloads:
-                        outfile.write("%s : %s\n" % (failed_id, products[failed_id]['title']))
+        product_infos, failed_downloads = api.download_all(products, path)
```

Comment:
Maybe keep the `checksum=True` here, to make the default behavior visible.

Reply:
I find that unnecessary -- specifying a keyword parameter giving it its default value.

Reply:
Yes, you can see it that way, too.